docs/api-and-integration/metadata/index.md

```diff
@@ -0,0 +1,120 @@
+
+If you have configured an issue tracker extension (e.g. Jira), that matches the work-items type, the work-items appear as a link to the issue tracking system. If not, the work-items appear as plain text.
+
+## Project Settings
+
+The next step is to tell your Octopus project which packages in its deployment process contain metadata that you want included in the releases and deployments.
+
+![Octopus Project work-item settings](octo-project.png)
+
+In this example, the project is using the primary package from the step named Deploy. You can specify multiple packages if required. An example of where you might use this is a deployment process that includes steps for a Web App and an SQL database schema migration. The packages for both of these form the complete product and either or both can contribute work-items.
+
+## Releases and Deployments
+
+When a release is created for the project, the work-items are collected from the selected package versions and displayed with the release details.
+
+![Release work-items](release-work-items.png)
+
+When this release is deployed the metadata from the release will be passed to the deployment. This metadata will be displayed in the deployment preview.
+
+![Deployment preview](deploy-preview-work-items.png)
+
+They are also displayed on the task summary for the deployment.
+
+![Deployment work-items](deploy-work-items.png)
+
+In some scenarios this one to one between releases and deployments will be the norm. Teams using continuous deployment may see a rapid progression of versions through their environments and on to production.
+
+Many teams don't operate like this though. It is common for a number of releases to be prepared and tested before being promoting to the next environment. For example, they may merge pull requests for several fixes/features and then deploy the final release to test. This can be repeated several times and eventually a final release progresses from test to production. In this situation the deployment to production isn't just the work-items from that last release, it is _the accumulation of all of the work-items in all of the releases since the last one that was deployed to production_.
+
+This accumulation logic is how Octopus always determines the metadata list, it's always the accumulation since the last deployment to the given "scope". A scope in this context is the combination of deployment environment and tenant (if [multi-tenancy](/docs/deployment-patterns/multi-tenant-deployments/index.md) is in play).
+
+Given that you can initiate deployments to multiple "scopes" at once in Octopus it is expected that you could see different work-items lists for each scope. 
+
+For instance, if you have two tenants in the same environment, and **tenant A** is on an earlier version than **tenant B**, the resulting list for **tenant A** will be the list for **tenant B** plus the additional work-items between the version it was on and the version **tenant B** was on.
```

Comment:
If I'm reading this correctly, I have a feeling we should make it clear this happens when we create a new release of tentant A and deploy it so that it is now on the same version as tenant B. The suggestion below attempts to make that clear.

```suggestion
For instance, if you have two tenants in the same environment, and **tenant A** is on an earlier version than **tenant B**, when **tenant A** moves to the same version as **tenant B**, the resulting list of work-items for **tenant A** will be the list for **tenant B** plus the additional work-items between the version **tenant A** was on and the version **tenant B** was on.
```